app/tasks/ssl.php

```diff
@@ -11,13 +11,15 @@
     ->action(function () {
         $domain = App::getEnv('_APP_DOMAIN', '');
 
-        Console::log('Issue a TLS certificate for master domain ('.$domain.') in 30 seconds.
+        // TODO: Instead of waiting, let's ping Traefik. If responds, we can schedule instantly
+        // TODO: Add support for argument (domain)
+
+        Console::log('Issue a TLS certificate for master domain ('.$domain.') in 2 seconds.
             Make sure your domain points to your server or restart to try again.');
 
-        ResqueScheduler::enqueueAt(\time() + 30, 'v1-certificates', 'CertificatesV1', [
-            'document' => [],
+        // Const for types not available here
+        ResqueScheduler::enqueueAt(\time() + 2, 'v1-certificates', 'CertificatesV1', [
             'domain' => $domain,
-            'validateTarget' => false,
-            'validateCNAME' => false,
+            'skipRenewCheck' => true // TODO: Discuss this behabiour. true? false? parameter? How do we document it?
```

Comment:
This is mainly a developer experience concern. If I directly run command to renew certificate.. Should we forcefully renew it, or tell him "Renew not required"?

The idea is if anything goes wrong, this command would ALWAYS renew a certificate, no matter what... It would be a quick fix for devs, and they can, later on, check logs to prevent issues in future.

(that was my thinking when implementing it)

Reply:
We decided to skip the check from CLI. This is an administration tool and should do exactly what it does, no unnecessary checks. We risk someone hit limit of 5 requests per week per successful domain, but it would be i responsible usage of the command. This will be all documented in docs.